### PR TITLE
Adds KCS-3

### PR DESCRIPTION
**Pull request by sgerbino**
Adds the already finalized KCS-3 standard. This is a modified version of the PR by @joticajulian.

### CODE REVIEW

File: KCSs/kcs-3.md [hunk 1]
```diff
@@ -0,0 +1,310 @@
...
+message transfer_args {
+   bytes from = 1 [(koinos.btype) = ADDRESS];
+   bytes to = 2 [(koinos.btype) = ADDRESS];
+   uint64 value = 3 [jstype = JS_STRING];
+}
+// Result
+message transfer_result {}
+```
+
+The transfer event should emit a `token_transfer` event upon success. The event should indicate the sender and receiver as impacted accounts.
```
Comment (joticajulian) [inline KCSs/kcs-3.md]: "should emit a `transfer_event` upon success."

(to be aligned with mint and burn events)

Comment (joticajulian) [inline KCSs/kcs-3.md]: > The event should indicate the sender and receiver as impacted accounts.

maybe indicate the order of impacted accounts? first one is receiver, second one is sender.

File: KCSs/kcs-3.md [hunk 2]
```diff
@@ -0,0 +1,310 @@
...
+
+```proto
+// Arguments
+message approve_args {
+   bytes owner = 1 [(koinos.btype) = ADDRESS];
+   bytes spender = 2 [(koinos.btype) = ADDRESS];
+   uint64 value = 3 [jstype = JS_STRING];
+}
+// Result
+message approve_result {}
```
Comment (joticajulian) [inline KCSs/kcs-3.md:291]: add an event

**Comment by joticajulian** [comment]
Thanks for working on this @sgerbino 👍

File: KCSs/kcs-3.md (hunk 2 above)
Comment (sgerbino) [reply KCSs/kcs-3.md:291]: We should add these standards/events in the `koinos-proto` repo and align them with standards. I'll look at your event in your token example.

Comment (sgerbino) [reply KCSs/kcs-3.md:291]:
```proto
message approve_event {
   bytes owner = 1 [(koinos.btype) = ADDRESS];
   bytes spender = 2 [(koinos.btype) = ADDRESS];
   uint64 value = 3 [jstype = JS_STRING];
}
```

Seems reasonable to me.

Comment (sgerbino) [reply KCSs/kcs-3.md:291]: Updated.

**Review by joticajulian — APPROVED** [review]

**Comment by sgerbino** [comment]
All protocol buffer method arguments should be `_arguments` and all results should be `_result`. Also, we should add each of these standards to https://github.com/koinos/koinos-proto. I propose we put them in `contracts/standard/kcs-*.proto`. I did not completely review all of these refactors, simply normalize them so they can be reviewed individually.

**Review by mvandeberg — APPROVED** [review]

**Pull request merged**